scripts/build/create_commands.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+"""
+Copy CodeChecker entry point sub-commands.
+"""
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+
+import argparse
+import json
+import logging
+import os
+import shutil
+
+LOG = logging.getLogger('EntryPoints')
+
+msg_formatter = logging.Formatter('[%(levelname)s] - %(message)s')
+log_handler = logging.StreamHandler()
+log_handler.setFormatter(msg_formatter)
+LOG.setLevel(logging.INFO)
+LOG.addHandler(log_handler)
+
+
+def copy_entry_points(input_dirs, build_dir):
+    """
+    Copy CodeChecker entry point sub-commands.
+    """
+    package_root = os.path.join(build_dir, 'CodeChecker')
+    package_bin = os.path.join(package_root, 'bin')
+    target_cc = os.path.join(package_root, 'cc_bin')
+
+    available_commands = []
+    for input_dir in input_dirs:
+        for _, _, files in os.walk(input_dir):
+            for f in files:
```

Comment:
using the [glob](https://docs.python.org/3.7/library/glob.html) module would not simplify this file filtering part?